src/components/ui/form-range/FormRange.vue

```diff
@@ -1,24 +1,39 @@
 <script setup lang="ts">
-import { computed, getCurrentInstance, onMounted, ref } from 'vue'
+import { computed, getCurrentInstance, onMounted, ref, type StyleValue } from 'vue'
 
 const props = defineProps<{
-	modelValue?: undefined
-	label?: string
+	modelValue?: string | number
 	required?: boolean
 	name?: string
 	min?: string | number
 	max?: string | number
 	step?: string | number
+	width?: string | number
+	disabled?: boolean
 }>()
 
 const uid = `ui-form-range-${getCurrentInstance()?.uid}`
-const slider = computed(() => {
-	return props.modelValue
-})
 const showBubble = ref(false)
 
 const emit = defineEmits(['update:modelValue'])
 
+const slider = computed<string>({
+	get() {
+		return props.modelValue as string
+	},
+	set(value: string) {
```

Comment:
Pelo que vi do código desse componente não existe nenhum lugar onde é feita uma alteração de valor da const computed slider, se n eu não tiver enganado não precisa desse setter

Reply:
Esse setter é para quem for usar o componente conseguir utilizar o syntax sugar  `v-model` que espera que o componente envie o update =]